operators/video_encoder/video_encoder_response/python/video_encoder_response_pydoc.hpp

```diff
@@ -0,0 +1,82 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef PYHOLOHUB_OPERATORS_VIDEO_ENCODER_RESPONSE_PYDOC_HPP
+#define PYHOLOHUB_OPERATORS_VIDEO_ENCODER_RESPONSE_PYDOC_HPP
+
+#include <string>
+
+#include "macros.hpp"
+
+namespace holoscan::doc {
+
+namespace VideoEncoderResponseOp {
+
+PYDOC(VideoEncoderResponseOp, R"doc(
+Operator class to perform inference using an LSTM model.
+)doc")
+
+// PyVideoEncoderResponseOp Constructor
+PYDOC(VideoEncoderResponseOp_python, R"doc(
```

Comment:
recently, for the SDK we changed to just using the full docstring version on both class and init method.

```suggestion
// PyVideoEncoderResponseOp Constructor
PYDOC(VideoEncoderResponseOp, R"doc(
```